src/elisp/treemacs-customization.el

```diff
@@ -740,6 +746,11 @@ fallback."
   :type 'boolean
   :group 'treemacs-follow)
 
+(defcustom treemacs-move-files-by-mouse-dragging t
+  "When non-nil `treemacs--drag-move-files' will move files by dragging with your mouse inside treemacs."
```

Comment:
This line needs to be less than 80 characters so the build will pass.

Your commits should also be squashed when you're done.